conversion/conversion.go

```diff
@@ -520,11 +534,11 @@ func dataFromDatabaseForDataflowMigration(targetProfile profiles.TargetProfile,
 
 	// create monitoring aggregated dashboard for sharded migration
 	aggMonitoringResources := metrics.MonitoringMetricsResources{
-		ProjectId:                     targetProfile.Conn.Sp.Project,
```

Comment:
nit: is this an accidental change?

Reply:
This the gofmt which formats the file, there is no code change just the indentation. Im guessing the setting is different for whoever merged this.